CMakeLists.txt

```diff
@@ -16,24 +16,30 @@ set(GCF_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/git-cmake-format.py)
 
 find_package(ClangFormat ${GCF_CLANGFORMAT_MINIMAL_VERSION} REQUIRED)
 
-execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --show-toplevel
+execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --absolute-git-dir
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
   OUTPUT_VARIABLE GCF_GIT_ROOT
   OUTPUT_STRIP_TRAILING_WHITESPACE)
 if(NOT GCF_GIT_ROOT)
   message(WARNING "Not in a git repository")
 else()
+  set(GCF_SCRIPT_GIT_DIR ${GCF_GIT_ROOT}/hooks/git-cmake-format.py)
```

Comment:
wrong name?

Reply:
Yes, it is not ideal, perhaps `GCF_SCRIPT_TARGET_DIR`?

Reply:
maybe GCF_GIT_DIR_SCRIPT, GCF_GIT_SCRIPT or something else?
I use the last term to identify the string type. I do not know it makes sense or not

Reply:
Your first one works fine, I will update it.